drivers/hmis_external_apis/app/models/hmis_external_apis/tc_hmis/importers/loaders/custom_data_element_helper.rb

```diff
@@ -6,17 +6,17 @@
 
 module HmisExternalApis::TcHmis::Importers::Loaders
   class CustomDataElementHelper
-    attr_accessor :data_source, :system_user, :today
+    attr_accessor :data_source_id, :system_user_id, :today
 
-    def initialize(data_source:, system_user:, today:)
-      self.data_source = data_source
-      self.system_user = system_user
+    def initialize(data_source_id:, system_user_id:, today:)
```

Comment:
So this loader is used by our migration process. I'd prefer not to make changes to this code while we are in middle of data migration unless there a bug or some issue this addresses. If this change isn't necessary, can you back it out (and the change to BaseLoader)? Thanks!